contentful/apps
# [INTEG-1068] AICG config screen brand profile section

## Purpose

This PR adds the brand profile fields on the config screen for AI Content Generator. 

<img width="935" alt="Screenshot 2023-08-16 at 11 58 35 AM" src="https://github.com/contentful/apps/assets/62958907/6d3d4b45-f593-4749-9db6-45c488e8f985">

## Approach

Added new fields on the config screen according to the designs. I also updated the `useSaveConfigHandler` hook to reduce the calls to `sdk.app.getCurrentState()`. I also updated some tests, more will be added later 😄 This PR does not sync up the new brand profile info to the prompts yet, that will be addressed in a follow up PR.

## Testing steps

Navigate to the AI Content Generator app config page to see the new fields. Add some text, save the config, and the inputs should be persisted in the app installation parameters.
